Makefile.am

```diff
@@ -842,13 +842,13 @@ ha_m0ham_CPPFLAGS  = -DM0_TARGET='m0ham' $(AM_CPPFLAGS)
 ha_m0ham_LDADD     = $(top_builddir)/motr/libmotr.la
 
 #
-# fdmi/plugins/m0sched -------------------------------------------- {{{2
+# fdmi/plugins/fdmi_sample_plugin --------------------- {{{2
 #
 
-sbin_PROGRAMS     += fdmi/plugins/m0sched
-sbin_SCRIPTS     += fdmi/plugins/m0scheduler
-fdmi_plugins_m0sched_CPPFLAGS  = -DM0_TARGET='m0sched' $(AM_CPPFLAGS)
-fdmi_plugins_m0sched_LDADD     = $(top_builddir)/motr/libmotr.la
+sbin_PROGRAMS     += fdmi/plugins/fdmi_sample_plugin
+sbin_SCRIPTS     += fdmi/plugins/fdmi_app
+fdmi_plugins_fdmi_sample_plugin_CPPFLAGS  = -DM0_TARGET='fdmi_sample_plugin' $(AM_CPPFLAGS)
+fdmi_plugins_fdmi_sample_plugin_LDADD     = $(top_builddir)/motr/libmotr.la
```

Comment:
Can you please change the target branch to "fdmi_int"?

So, the "fdmi/plugins/Makefile.sub" should be added/updated.

Reply:
What do you mean change branch? Eventually we will do it but how is it related to makefiles?

Reply:
Please make sure to merge this PR to fdmi_int branch. Please refer to #731.

Reply:
sorry but I still not following. How #731 is related? It is about increasing buff size for messages.

Reply:
I am sorry, I mean, #719